content/concepts/loaders.md

```diff
@@ -60,8 +60,7 @@ There are three ways to use loaders in your application:
 ### Via `webpack.config.js`
 
 [`module.rules`](https://webpack.js.org/configuration/module/#module-rules) allows you to specify several loaders within your webpack configuration.
-This is a concise way to display loaders, and helps to have clean code as
-well as you have a full overview of each respective loader.
+This is a concise way to display loaders, and helps to maintain clean code. It also offers you a full overview of each respective loader. If you would like to enable css modules, set `modules: true.` Otherwise, be sure to set `modules: false`.  
```

Comment:
There are already two links in the [example section above](https://webpack.js.org/concepts/loaders/#example). However, I do not object a third link in the other section if that helps. For consistency, please also add a link for `style-loader` in the same example.